steve/scrapers.py

```diff
@@ -27,7 +25,7 @@ class YoutubeScraper(object):
     def transform_item(self, item):
         """Converts youtube-dl output to richard fields"""
         return {
-            'title': item['fulltitle'],
+            'title': item['title'],
```

Comment:
Are there docs where they list the elements they return? I don't know why some of these fields are changing.